### DISCUSSION TITLE
Cannot change base branch of a PR.

### DISCUSSION THREAD

**Discussion by Petersdavis** [community/community#60337] [Pull Requests]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

When we create a PR we can select the base branch-- but when editing the list of available branches does not populate:

![image](https://github.com/community/community/assets/19160989/2d5d814b-502e-45fc-9b33-dae709c3c9a5)

clicking on the branch does nothing.
no error in console.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Petersdavis ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
